frontend/src/scenes/trends/personsModalLogic.ts

```diff
@@ -320,7 +320,7 @@ export const personsModalLogic = kea<personsModalLogicType>({
                     const funnelParams = toParams(cleanedParams)
                     let includeRecordingsParam = ''
                     if (values.featureFlags[FEATURE_FLAGS.RECORDINGS_IN_INSIGHTS]) {
-                        includeRecordingsParam = 'include_recordings=true&'
+                        includeRecordingsParam = '&include_recordings=true'
```

Comment:
this looks a bit weird, you sure this works?

Since: `api/person/funnel/?${includeRecordingsParam}` - recordingsParam is first?

Reply:
That still works but the other way round doesn't as all other have the `&` in front

Reply:
Ahh, makes sense

Reply:
This will also be tidied in PersonsModalV2